src/RwaUrnWipeJob.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
Would like to merge these two into one job. There is overhead for each job we add and the code between these two files is roughly identical. I would just check for both "jar" and "urn" components at the same time and execute either if necessary.

Reply:
This can be a little problematic in the future because when we have different components requiring keeper jobs we'd need to update this or create a new one and this can become messy.

These are the 2 "easiest" jobs that we can think of, but for sure there will be more complex ones in the future, so we rather keep the heuristic of having 1 job per each operation.

Reply:
We could create some sort of base RWARegistry job, which takes care of common things ( checking if a components exists, if the component address and is not `address(0)`, etc.), but I'd still keep the "core" logic separate for each type of action.